guides/plugins/plugins/framework/custom-field/add-custom-field.md

```diff
@@ -244,6 +244,7 @@ use \Shopware\Core\Defaults;
 $this->customFieldSetRepository->create([
     [
         'name' => 'swag_example_set',
+        'global' => true,
```

Comment:
@JoshuaBehrens I would say we keep one or the other. So add the global => true and remove the warning.

When adding the customField via Code, I suppose it is 99% unwanted to be deleted via the admin.

We could also change the warning to an info and say:

_"If you want the custom field set to be deleteable and editable in the administration, you need to set global to false"_